testing/src/test/java/org/oppia/android/testing/espresso/TextInputActionTest.kt

```diff
@@ -0,0 +1,96 @@
+package org.oppia.android.testing.espresso
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.android.material.textfield.TextInputLayout
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.testing.TextInputActionTestActivity
+import org.oppia.android.testing.espresso.TextInputAction.Companion.ErrorTextExisted
+import org.robolectric.annotation.LooperMode
+
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+class TextInputActionTest {
+
+  @get:Rule
+  var activityRule =
+    ActivityScenarioRule<TextInputActionTestActivity>(
+      TextInputActionTestActivity.createIntent(ApplicationProvider.getApplicationContext())
+    )
+
+  @Test
+  fun testTextInputAction_ErrorTextExisted_matchesSafelyTrue() {
+    lateinit var context: Context
+    activityRule.scenario.onActivity {
+      context = it.baseContext
+      // context = it.applicationContext
+    }
+
+    val expectedErrorText = "Incorrect Administrator PIN. Please try again."
+    val textInputLayout = TextInputLayout(context)
+
```

Comment:
@BenHenning Sorry for being inactive on this lately. I am not really sure about this approach for passing context to ```TextInputLayout``` but I anyway tried going in this direction but I am getting the same error this way as well. I also tried programmatically creating a ```TextInputLayout``` in the ```TextInputActionTestActivity.kt``` itself but I am unable to reference and use it in the test file. Any suggestions?

Reply:
@ayush0402 what issue are you running into with this approach? It seems on the correct track, so I'm not sure what problem you need help with. Can you please clarify what exactly isn't working?

Reply:
@BenHenning This was giving the following error : 
```
/home/ayush/android-studio/jre/bin/java -ea -Didea.test.cyclic.buffer.size=1048576 -javaagent:/home/ayush/android-studio/lib/idea_rt.jar=44559:/home/ayush/android-studio/bin -Dfile.encoding=UTF-8 -classpath /home/ayush/android-studio/lib/idea_rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit5-rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit-rt.jar:/home/ayush/Android/Sdk/platforms/android-29/data/res:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debugUnitTest/classes:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/unit_test_config_directory/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/debugUnitTest/R.jar:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.20/411fc46e908bfa9c034f52b0d31b2e1f61f06127/kotlin-reflect-1.4.20.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/jaxb-runtime/2.3.2/5528bc882ea499a09d720b42af11785c4fc6be2a/jaxb-runtime-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jvnet.staxex/stax-ex/1.8.1/78011e483a21102fb4858f3e8f269a677e50aa23/stax-ex-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.xml.bind/jakarta.xml.bind-api/2.3.2/8d49996a4338670764d7ca4b85a1c4ccf7fe665d/jakarta.xml.bind-api-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.4/418c5bfae392fdbf71cd463a42a3e8c3b839a924/robolectric-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.4/fbcda51d8e6f3a3897ae5cedc7aa481815745290/junit-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.4/3cedd73c5aedaf79fb9a593552816c9fb3282f2/sandbox-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.4/90028766e71353ad6f57d7bcb56ac0d861da18c3/shadows-framework-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.4/a2ee1324bcb62724e6cbfa655bdb5683948a554c/resources-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.4/44c40ac0d2ef1e7c8b0f6c4e224ef26d356170f1/utils-reflector-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.4/9241a3c4bd01627447c76d9b67614808c78ffdd9/plugins-maven-dependency-resolver-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.4/c54b2638d64e7bd4e1e45c4fe8038305402bd711/utils-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.4.20/8f6239ddd98bf9937a81d8a932a185dab930e82a/kotlin-android-extensions-runtime-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-test/1.2.2/93d934029985b6ae62c26cb7fcbdfeecd9b684be/kotlinx-coroutines-test-1.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.10/e2b3c6695eee6085e606d96d685396dce23a3a06/kotlin-reflect-1.4.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.3.4/21598c7ceabbcafb953cd151cd5ad44615a253c0/kotlinx-coroutines-core-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/converter-moshi/2.5.0/95fe2632a019f26d9c53129db192ebc7f3524d35/converter-moshi-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit-mock/2.5.0/3615f8951fd30235331a6027779dfae98b203132/retrofit-mock-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit/2.5.0/713ce36037bf24a76a3974c05cb85c3f754b1cc3/retrofit-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/4.1.0/9308be2b5f9174def7d4795f415b60f5632b9fb9/okhttp-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.21/4a668382d7c38688d3490afde93b6a113ed46698/kotlin-stdlib-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.21/7f48a062aa4b53215998780f7c245a4276828e1d/kotlin-stdlib-common-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.2.0-alpha03/4ef09a745007778eef83b92f8f23987a8ea59496/lifecycle-common-2.2.0-alpha03.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.1.0/e3a6fb2f40e3a3842e6b7472628ba4ce416ea4c8/annotation-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth/truth/1.1.3/5e2491cc44dce7380165963a24e91a0af4556a17/truth-1.1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.android.apps.common.testing.accessibility.framework/accessibility-test-framework/2.0/7837f95bdff9f797003454247fef02b78d49ae75/accessibility-test-framework-2.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-integration/1.3/5de0c73fef18917cd85d0ab70bb23818685e4dfd/hamcrest-integration-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-library/1.3/4785a3c21320980282f9f33d0d1264a69040538f/hamcrest-library-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.sf.kxml/kxml2/2.3.0/ccbc77a5fd907ef863c29f3596c6f54ffa4e9442/kxml2-2.3.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup/javawriter/2.1.1/67ff45d9ae02e583d0f9b3432a5ebbe05c30c966/javawriter-2.1.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.dagger/dagger/2.27/7bee2792b10523e298a7de508635c6eed0c309be/dagger-2.27.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/30.1.1-android/5963ed171d561cca6f14659f3439b46a1633ab13/guava-30.1.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protobuf-javalite/3.17.3/a07accfe99ff27e871490cf29fb0d8b0249de320/protobuf-javalite-3.17.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-compat-qual/2.5.5/435dc33e3019c9f019e15f01aa111de9d6b2b79c/checker-compat-qual-2.5.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.7.1/458d9042f7aa6fa9a634df902b37f544e15aacac/error_prone_annotations-2.7.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.3/ba035118bc8bac37d7eff77700720999acd9986d/j2objc-annotations-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/3.13.0/e29afc6adb63a39286d38dc0e00d164117c02f0d/checker-qual-3.13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.8.1/174878f9cc9cbacb8b464d774afb2ccf938f257b/auto-value-annotations-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/7.2/ca2954e8d92a05bacc28ff465b25c70e0f512497/asm-commons-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/7.2/a3ae34e57fa8a4040e28247291d0cc3d6b8c7bcf/asm-util-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/7.2/b6e6abe057f23630113f4167c34bda7086691258/asm-analysis-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/7.2/3a23cc36edaf8fc5a89cb100182758ccb5991487/asm-tree-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/9.1/a99500cf6eea30535eeac6be73899d048f8d12a8/asm-9.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.4/1ee94260f8c51620a35eac33fc1efc01350c751f/pluginapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.4/529649474b53cf8f6f4a483044ade43aebed8a4c/shadowapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.4/70fc5b1699467dfd7de606fc6c02ff9fc1816d9f/annotations-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/0.282/745a7e2f35fdbe6336922e0d492c979dbbfa74fb/sqlite4java-0.282.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.65/320b989112f00a63a3bcfa5a98f31a4f865a20fa/bcprov-jdk15on-1.65.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.19.0/bc51dab3138afbd8517511cfb183dab731d8b1c4/mockito-core-2.19.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.8.10/c3dcdb62b4297eacbe8a763f36044e1f1b04709b/byte-buddy-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.8.10/957933bd13319beb33138494122e8fd1c347bce4/byte-buddy-agent-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/2.6/639033469776fd37c08358c6b92a4761feb2af4b/objenesis-2.6.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi/1.11.0/14c426fc49ee29eb871e8b259326e967082a72ad/moshi-1.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/2.2.2/36f483536153f15339a8b48d508e22be7c9c531a/okio-2.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.istack/istack-commons-runtime/3.0.8/d6a97364045aa6b99bf2d3c566a3f98599c2d296/istack-commons-runtime-3.0.8.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.activation/jakarta.activation-api/1.2.1/562a587face36ec7eff2db7f2fc95425c6602bc1/jakarta.activation-api-1.2.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/txw2/2.3.2/ce5be7da2e442c25ec14c766cb60cb802741727b/txw2-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.xml.fastinfoset/FastInfoset/1.2.16/4eb6a0adad553bf759ffe86927df6f3b848c8bea/FastInfoset-1.2.16.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/res:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/res:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/domain/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/domain/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/domain/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/domain/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/mockwebserver/4.1.0/7269c5f584bef7851fbfb38cd9c9be5959ceae79/mockwebserver-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-junit/1.4.20/e071a8fda380cb9d347a40e920febab251b464e2/kotlin-test-junit-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test/1.4.20/37f4d139996ed0bbd88df805e4df833040431ff6/kotlin-test-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-annotations-common/1.4.20/a82b48a34377ac287c138ea21e09b511ab21c374/kotlin-test-annotations-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-common/1.4.20/2191d43275c9a8558adea84729c1ca372a463e59/kotlin-test-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth.extensions/truth-liteproto-extension/1.1.3/b293a67586f9a39d9f1e0f0ed3174c180ec7070/truth-liteproto-extension-1.1.3.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.20/9de2c79e95d4b4699a455e88ba285a95352e0bea/kotlin-stdlib-jdk7-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.3.4/93dc6aca51305c1b58ccf5a6811665ed58f48ff1/kotlinx-coroutines-android-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.20/9be77b243a362b745e365f286627b8724337009c/kotlin-stdlib-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.20/c6761d7805b5312302f2bbd78cda68c976ce0c70/kotlin-stdlib-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.0.0/42858b26cafdaa69b6149f45dfc2894007bc2c7a/collection-1.0.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/28.1-android/c2526f8fad32a65a6d7032dd8e9524eb276b108b/guava-28.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.3.2/d1a0c5032570e0f64be6b4d9c90cdeb103129029/error_prone_annotations-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.18/f7aa683ea79dc6681ee9fb95756c999acbb62f5d/animal-sniffer-annotations-1.18.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.6.5/c3dad10377f0e2242c9a4b88e9704eaf79103679/auto-value-annotations-1.6.5.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/res:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/res:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/data/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/data/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/data/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/data/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/.gradle/caches/transforms-3/c0b72f579d9d1fe784b93659eeaa0248/transformed/multidex-instrumentation-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2470781e78212e0a3fca6abd7b631a19/transformed/monitor-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi-kotlin/1.11.0/d8f796170ae085954c9185351a831e276a3d44/moshi-kotlin-1.11.0.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/utility/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/utility/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/utility/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/utility/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/1.0/c949a840a6acbc5268d088e47b04177bf90b3cad/listenablefuture-1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.oppia/androidsvg/6bd15f69caee3e6857fcfcd123023716b4adec1d/58890ad245ff948ce9f19c772dcb88dbf5b8544e/androidsvg-6bd15f69caee3e6857fcfcd123023716b4adec1d.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/disklrucache/4.11.0/ed93d2e20549ad85f692d964788ec77520d78a8b/disklrucache-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/annotations/4.11.0/c57bae5a18147f8ae22f4da49baac875c6b6f84f/annotations-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.12.1/dc6d02e4e68514eff5631963e28ca7742ac69efe/okhttp-3.12.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/1.15.0/bc28b5a964c8f5721eb58ee3f3c47a9bcbf4f4d8/okio-1.15.0.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/res:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/model/build/classes/java/main:/home/ayush/opensource/oppia-android/model/build/resources/main:/home/ayush/opensource/oppia-android/model/build/libs/model.jar:/home/ayush/.gradle/caches/transforms-3/bb2c0feeabbe4ca138704f9cfbac2aab/transformed/room-runtime-2.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9399f18d7867437e16131879faa6bcbf/transformed/jetified-lifecycle-service-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/97dfe740135addea92e170a34026ecc8/transformed/sqlite-framework-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6e04fd8affb200758671fe93167571d0/transformed/sqlite-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.2.5/f5e3b73a0c2ab5e276e26868e4ce3542baede207/room-common-2.2.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-debug/1.2.2/d24ea2861c7b84965b0e395f6c09ffedf27f890d/kotlinx-coroutines-debug-1.2.2.jar:/home/ayush/opensource/oppia-android/data/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/domain/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/utility/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/model/build/resources/test:/home/ayush/.gradle/caches/transforms-3/c4c90c1865fbc502b00d7cf6afab03c8/transformed/android.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 org.oppia.android.testing.espresso.TextInputActionTest,testTextInputAction_ErrorTextExisted_matchesSafelyTrue
[Robolectric] WARN: Android SDK 29 requires Java 9 (have Java 8). Tests won't be run on SDK 29 unless explicitly requested.
Called loadFromPath(/system/framework/framework-res.apk, true); mode=binary sdk=28

java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.

	at androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor(AppCompatDelegateImpl.java:696)
	at androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor(AppCompatDelegateImpl.java:659)
	at androidx.appcompat.app.AppCompatDelegateImpl.onPostCreate(AppCompatDelegateImpl.java:376)
	at androidx.appcompat.app.AppCompatActivity.onPostCreate(AppCompatActivity.java:118)
	at org.robolectric.util.ReflectionHelpers$6.run(ReflectionHelpers.java:254)
	at org.robolectric.util.ReflectionHelpers.traverseClassHierarchy(ReflectionHelpers.java:425)
	at org.robolectric.util.ReflectionHelpers.callInstanceMethod(ReflectionHelpers.java:248)
	at org.robolectric.android.controller.ComponentController$1.run(ComponentController.java:53)
	at org.robolectric.shadows.ShadowPausedLooper.runPaused(ShadowPausedLooper.java:195)
	at org.robolectric.android.controller.ComponentController.invokeWhilePaused(ComponentController.java:50)
	at org.robolectric.android.controller.ActivityController.postCreate(ActivityController.java:144)
	at org.robolectric.android.fakes.RoboMonitoringInstrumentation.startActivitySyncInternal(RoboMonitoringInstrumentation.java:71)
	at org.robolectric.android.internal.LocalActivityInvoker.startActivity(LocalActivityInvoker.java:40)
	at androidx.test.core.app.ActivityScenario.launchInternal(ActivityScenario.java:231)
	at androidx.test.core.app.ActivityScenario.launch(ActivityScenario.java:209)
	at androidx.test.ext.junit.rules.ActivityScenarioRule.lambda$new$1$ActivityScenarioRule(ActivityScenarioRule.java:77)
	at androidx.test.ext.junit.rules.ActivityScenarioRule$$Lambda$1.get(Unknown Source)
	at androidx.test.ext.junit.rules.ActivityScenarioRule.before(ActivityScenarioRule.java:82)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:50)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:575)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$0(SandboxTestRunner.java:263)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:89)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)


Process finished with exit code 255
```
I also tried adding ```"android:theme="@style/Theme.AppCompat.Light"``` in testing/AndroidManifest.xml for this activity (Adding it under the application tag in both modules did not help either) but then there was this error : 
```
/home/ayush/android-studio/jre/bin/java -ea -Didea.test.cyclic.buffer.size=1048576 -javaagent:/home/ayush/android-studio/lib/idea_rt.jar=34661:/home/ayush/android-studio/bin -Dfile.encoding=UTF-8 -classpath /home/ayush/android-studio/lib/idea_rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit5-rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit-rt.jar:/home/ayush/Android/Sdk/platforms/android-29/data/res:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debugUnitTest/classes:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/unit_test_config_directory/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/debugUnitTest/R.jar:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.20/411fc46e908bfa9c034f52b0d31b2e1f61f06127/kotlin-reflect-1.4.20.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/jaxb-runtime/2.3.2/5528bc882ea499a09d720b42af11785c4fc6be2a/jaxb-runtime-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jvnet.staxex/stax-ex/1.8.1/78011e483a21102fb4858f3e8f269a677e50aa23/stax-ex-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.xml.bind/jakarta.xml.bind-api/2.3.2/8d49996a4338670764d7ca4b85a1c4ccf7fe665d/jakarta.xml.bind-api-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.4/418c5bfae392fdbf71cd463a42a3e8c3b839a924/robolectric-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.4/fbcda51d8e6f3a3897ae5cedc7aa481815745290/junit-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.4/3cedd73c5aedaf79fb9a593552816c9fb3282f2/sandbox-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.4/90028766e71353ad6f57d7bcb56ac0d861da18c3/shadows-framework-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.4/a2ee1324bcb62724e6cbfa655bdb5683948a554c/resources-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.4/44c40ac0d2ef1e7c8b0f6c4e224ef26d356170f1/utils-reflector-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.4/9241a3c4bd01627447c76d9b67614808c78ffdd9/plugins-maven-dependency-resolver-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.4/c54b2638d64e7bd4e1e45c4fe8038305402bd711/utils-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.4.20/8f6239ddd98bf9937a81d8a932a185dab930e82a/kotlin-android-extensions-runtime-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-test/1.2.2/93d934029985b6ae62c26cb7fcbdfeecd9b684be/kotlinx-coroutines-test-1.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.10/e2b3c6695eee6085e606d96d685396dce23a3a06/kotlin-reflect-1.4.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.3.4/21598c7ceabbcafb953cd151cd5ad44615a253c0/kotlinx-coroutines-core-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/converter-moshi/2.5.0/95fe2632a019f26d9c53129db192ebc7f3524d35/converter-moshi-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit-mock/2.5.0/3615f8951fd30235331a6027779dfae98b203132/retrofit-mock-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit/2.5.0/713ce36037bf24a76a3974c05cb85c3f754b1cc3/retrofit-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/4.1.0/9308be2b5f9174def7d4795f415b60f5632b9fb9/okhttp-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.21/4a668382d7c38688d3490afde93b6a113ed46698/kotlin-stdlib-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.21/7f48a062aa4b53215998780f7c245a4276828e1d/kotlin-stdlib-common-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.2.0-alpha03/4ef09a745007778eef83b92f8f23987a8ea59496/lifecycle-common-2.2.0-alpha03.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.1.0/e3a6fb2f40e3a3842e6b7472628ba4ce416ea4c8/annotation-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth/truth/1.1.3/5e2491cc44dce7380165963a24e91a0af4556a17/truth-1.1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.android.apps.common.testing.accessibility.framework/accessibility-test-framework/2.0/7837f95bdff9f797003454247fef02b78d49ae75/accessibility-test-framework-2.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-integration/1.3/5de0c73fef18917cd85d0ab70bb23818685e4dfd/hamcrest-integration-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-library/1.3/4785a3c21320980282f9f33d0d1264a69040538f/hamcrest-library-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.sf.kxml/kxml2/2.3.0/ccbc77a5fd907ef863c29f3596c6f54ffa4e9442/kxml2-2.3.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup/javawriter/2.1.1/67ff45d9ae02e583d0f9b3432a5ebbe05c30c966/javawriter-2.1.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.dagger/dagger/2.27/7bee2792b10523e298a7de508635c6eed0c309be/dagger-2.27.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/30.1.1-android/5963ed171d561cca6f14659f3439b46a1633ab13/guava-30.1.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protobuf-javalite/3.17.3/a07accfe99ff27e871490cf29fb0d8b0249de320/protobuf-javalite-3.17.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-compat-qual/2.5.5/435dc33e3019c9f019e15f01aa111de9d6b2b79c/checker-compat-qual-2.5.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.7.1/458d9042f7aa6fa9a634df902b37f544e15aacac/error_prone_annotations-2.7.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.3/ba035118bc8bac37d7eff77700720999acd9986d/j2objc-annotations-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/3.13.0/e29afc6adb63a39286d38dc0e00d164117c02f0d/checker-qual-3.13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.8.1/174878f9cc9cbacb8b464d774afb2ccf938f257b/auto-value-annotations-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/7.2/ca2954e8d92a05bacc28ff465b25c70e0f512497/asm-commons-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/7.2/a3ae34e57fa8a4040e28247291d0cc3d6b8c7bcf/asm-util-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/7.2/b6e6abe057f23630113f4167c34bda7086691258/asm-analysis-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/7.2/3a23cc36edaf8fc5a89cb100182758ccb5991487/asm-tree-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/9.1/a99500cf6eea30535eeac6be73899d048f8d12a8/asm-9.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.4/1ee94260f8c51620a35eac33fc1efc01350c751f/pluginapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.4/529649474b53cf8f6f4a483044ade43aebed8a4c/shadowapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.4/70fc5b1699467dfd7de606fc6c02ff9fc1816d9f/annotations-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/0.282/745a7e2f35fdbe6336922e0d492c979dbbfa74fb/sqlite4java-0.282.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.65/320b989112f00a63a3bcfa5a98f31a4f865a20fa/bcprov-jdk15on-1.65.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.19.0/bc51dab3138afbd8517511cfb183dab731d8b1c4/mockito-core-2.19.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.8.10/c3dcdb62b4297eacbe8a763f36044e1f1b04709b/byte-buddy-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.8.10/957933bd13319beb33138494122e8fd1c347bce4/byte-buddy-agent-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/2.6/639033469776fd37c08358c6b92a4761feb2af4b/objenesis-2.6.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi/1.11.0/14c426fc49ee29eb871e8b259326e967082a72ad/moshi-1.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/2.2.2/36f483536153f15339a8b48d508e22be7c9c531a/okio-2.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.istack/istack-commons-runtime/3.0.8/d6a97364045aa6b99bf2d3c566a3f98599c2d296/istack-commons-runtime-3.0.8.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.activation/jakarta.activation-api/1.2.1/562a587face36ec7eff2db7f2fc95425c6602bc1/jakarta.activation-api-1.2.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/txw2/2.3.2/ce5be7da2e442c25ec14c766cb60cb802741727b/txw2-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.xml.fastinfoset/FastInfoset/1.2.16/4eb6a0adad553bf759ffe86927df6f3b848c8bea/FastInfoset-1.2.16.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/res:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/res:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/domain/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/domain/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/domain/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/domain/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/mockwebserver/4.1.0/7269c5f584bef7851fbfb38cd9c9be5959ceae79/mockwebserver-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-junit/1.4.20/e071a8fda380cb9d347a40e920febab251b464e2/kotlin-test-junit-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test/1.4.20/37f4d139996ed0bbd88df805e4df833040431ff6/kotlin-test-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-annotations-common/1.4.20/a82b48a34377ac287c138ea21e09b511ab21c374/kotlin-test-annotations-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-common/1.4.20/2191d43275c9a8558adea84729c1ca372a463e59/kotlin-test-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth.extensions/truth-liteproto-extension/1.1.3/b293a67586f9a39d9f1e0f0ed3174c180ec7070/truth-liteproto-extension-1.1.3.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.20/9de2c79e95d4b4699a455e88ba285a95352e0bea/kotlin-stdlib-jdk7-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.3.4/93dc6aca51305c1b58ccf5a6811665ed58f48ff1/kotlinx-coroutines-android-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.20/9be77b243a362b745e365f286627b8724337009c/kotlin-stdlib-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.20/c6761d7805b5312302f2bbd78cda68c976ce0c70/kotlin-stdlib-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.0.0/42858b26cafdaa69b6149f45dfc2894007bc2c7a/collection-1.0.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/28.1-android/c2526f8fad32a65a6d7032dd8e9524eb276b108b/guava-28.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.3.2/d1a0c5032570e0f64be6b4d9c90cdeb103129029/error_prone_annotations-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.18/f7aa683ea79dc6681ee9fb95756c999acbb62f5d/animal-sniffer-annotations-1.18.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.6.5/c3dad10377f0e2242c9a4b88e9704eaf79103679/auto-value-annotations-1.6.5.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/res:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/res:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/data/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/data/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/data/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/data/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/.gradle/caches/transforms-3/c0b72f579d9d1fe784b93659eeaa0248/transformed/multidex-instrumentation-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2470781e78212e0a3fca6abd7b631a19/transformed/monitor-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi-kotlin/1.11.0/d8f796170ae085954c9185351a831e276a3d44/moshi-kotlin-1.11.0.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/utility/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/utility/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/utility/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/utility/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/1.0/c949a840a6acbc5268d088e47b04177bf90b3cad/listenablefuture-1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.oppia/androidsvg/6bd15f69caee3e6857fcfcd123023716b4adec1d/58890ad245ff948ce9f19c772dcb88dbf5b8544e/androidsvg-6bd15f69caee3e6857fcfcd123023716b4adec1d.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/disklrucache/4.11.0/ed93d2e20549ad85f692d964788ec77520d78a8b/disklrucache-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/annotations/4.11.0/c57bae5a18147f8ae22f4da49baac875c6b6f84f/annotations-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.12.1/dc6d02e4e68514eff5631963e28ca7742ac69efe/okhttp-3.12.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/1.15.0/bc28b5a964c8f5721eb58ee3f3c47a9bcbf4f4d8/okio-1.15.0.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/res:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/model/build/classes/java/main:/home/ayush/opensource/oppia-android/model/build/resources/main:/home/ayush/opensource/oppia-android/model/build/libs/model.jar:/home/ayush/.gradle/caches/transforms-3/bb2c0feeabbe4ca138704f9cfbac2aab/transformed/room-runtime-2.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9399f18d7867437e16131879faa6bcbf/transformed/jetified-lifecycle-service-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/97dfe740135addea92e170a34026ecc8/transformed/sqlite-framework-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6e04fd8affb200758671fe93167571d0/transformed/sqlite-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.2.5/f5e3b73a0c2ab5e276e26868e4ce3542baede207/room-common-2.2.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-debug/1.2.2/d24ea2861c7b84965b0e395f6c09ffedf27f890d/kotlinx-coroutines-debug-1.2.2.jar:/home/ayush/opensource/oppia-android/data/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/domain/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/utility/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/model/build/resources/test:/home/ayush/.gradle/caches/transforms-3/c4c90c1865fbc502b00d7cf6afab03c8/transformed/android.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 org.oppia.android.testing.espresso.TextInputActionTest,testTextInputAction_ErrorTextExisted_matchesSafelyTrue
[Robolectric] WARN: Android SDK 29 requires Java 9 (have Java 8). Tests won't be run on SDK 29 unless explicitly requested.
Called loadFromPath(/system/framework/framework-res.apk, true); mode=binary sdk=28

java.lang.IllegalArgumentException: The style on this component requires your app theme to be Theme.AppCompat (or a descendant).

	at com.google.android.material.internal.ThemeEnforcement.checkTheme(ThemeEnforcement.java:243)
	at com.google.android.material.internal.ThemeEnforcement.checkAppCompatTheme(ThemeEnforcement.java:213)
	at com.google.android.material.internal.ThemeEnforcement.checkCompatibleTheme(ThemeEnforcement.java:148)
	at com.google.android.material.internal.ThemeEnforcement.obtainTintedStyledAttributes(ThemeEnforcement.java:115)
	at com.google.android.material.textfield.TextInputLayout.<init>(TextInputLayout.java:451)
	at com.google.android.material.textfield.TextInputLayout.<init>(TextInputLayout.java:410)
	at com.google.android.material.textfield.TextInputLayout.<init>(TextInputLayout.java:406)
	at org.oppia.android.testing.espresso.TextInputActionTest.testTextInputAction_ErrorTextExisted_matchesSafelyTrue(TextInputActionTest.kt:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:575)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$0(SandboxTestRunner.java:263)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:89)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)


Process finished with exit code 255
```

Reply:
@ayush0402 I think the second error is because Gradle can't use test manifests (it's one of the reasons we're moving to Bazel). You could try setting up a test application (we have examples of test applications in other test files) and use something like https://stackoverflow.com/a/46850678/3689782 to force the theme. It's a bit hacky, but a lot of this setup stuff is awkward at the moment because of inconsistencies between Gradle & Bazel, so I'm fine with a bare-bones solution that works.

Reply:
@BenHenning This [answer](https://stackoverflow.com/a/65318299/14822653) under the same question on StackOverflow seems to be a much simpler approach and has solved the problems with context. 

_Extra background :_

This approach could have also had worked with ```ApplicationProvider.getApplicationContext()``` (without TestActivity) directly but was giving out this error in case the test fails : 
```
/home/ayush/android-studio/jre/bin/java -ea -Didea.test.cyclic.buffer.size=1048576 -javaagent:/home/ayush/android-studio/lib/idea_rt.jar=36517:/home/ayush/android-studio/bin -Dfile.encoding=UTF-8 -classpath /home/ayush/android-studio/lib/idea_rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit5-rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit-rt.jar:/home/ayush/Android/Sdk/platforms/android-29/data/res:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debugUnitTest/classes:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/debugUnitTest/R.jar:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/unit_test_config_directory/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debug:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.20/411fc46e908bfa9c034f52b0d31b2e1f61f06127/kotlin-reflect-1.4.20.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/jaxb-runtime/2.3.2/5528bc882ea499a09d720b42af11785c4fc6be2a/jaxb-runtime-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jvnet.staxex/stax-ex/1.8.1/78011e483a21102fb4858f3e8f269a677e50aa23/stax-ex-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.xml.bind/jakarta.xml.bind-api/2.3.2/8d49996a4338670764d7ca4b85a1c4ccf7fe665d/jakarta.xml.bind-api-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.4/418c5bfae392fdbf71cd463a42a3e8c3b839a924/robolectric-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.4/fbcda51d8e6f3a3897ae5cedc7aa481815745290/junit-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.4/3cedd73c5aedaf79fb9a593552816c9fb3282f2/sandbox-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.4/90028766e71353ad6f57d7bcb56ac0d861da18c3/shadows-framework-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.4/a2ee1324bcb62724e6cbfa655bdb5683948a554c/resources-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.4/44c40ac0d2ef1e7c8b0f6c4e224ef26d356170f1/utils-reflector-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.4/9241a3c4bd01627447c76d9b67614808c78ffdd9/plugins-maven-dependency-resolver-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.4/c54b2638d64e7bd4e1e45c4fe8038305402bd711/utils-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.4.20/8f6239ddd98bf9937a81d8a932a185dab930e82a/kotlin-android-extensions-runtime-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-test/1.2.2/93d934029985b6ae62c26cb7fcbdfeecd9b684be/kotlinx-coroutines-test-1.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.10/e2b3c6695eee6085e606d96d685396dce23a3a06/kotlin-reflect-1.4.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.3.4/21598c7ceabbcafb953cd151cd5ad44615a253c0/kotlinx-coroutines-core-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/converter-moshi/2.5.0/95fe2632a019f26d9c53129db192ebc7f3524d35/converter-moshi-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit-mock/2.5.0/3615f8951fd30235331a6027779dfae98b203132/retrofit-mock-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit/2.5.0/713ce36037bf24a76a3974c05cb85c3f754b1cc3/retrofit-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/4.1.0/9308be2b5f9174def7d4795f415b60f5632b9fb9/okhttp-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.21/4a668382d7c38688d3490afde93b6a113ed46698/kotlin-stdlib-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.21/7f48a062aa4b53215998780f7c245a4276828e1d/kotlin-stdlib-common-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.2.0-alpha03/4ef09a745007778eef83b92f8f23987a8ea59496/lifecycle-common-2.2.0-alpha03.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.1.0/e3a6fb2f40e3a3842e6b7472628ba4ce416ea4c8/annotation-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth/truth/1.1.3/5e2491cc44dce7380165963a24e91a0af4556a17/truth-1.1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.android.apps.common.testing.accessibility.framework/accessibility-test-framework/2.0/7837f95bdff9f797003454247fef02b78d49ae75/accessibility-test-framework-2.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-integration/1.3/5de0c73fef18917cd85d0ab70bb23818685e4dfd/hamcrest-integration-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-library/1.3/4785a3c21320980282f9f33d0d1264a69040538f/hamcrest-library-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.sf.kxml/kxml2/2.3.0/ccbc77a5fd907ef863c29f3596c6f54ffa4e9442/kxml2-2.3.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup/javawriter/2.1.1/67ff45d9ae02e583d0f9b3432a5ebbe05c30c966/javawriter-2.1.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.dagger/dagger/2.27/7bee2792b10523e298a7de508635c6eed0c309be/dagger-2.27.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/30.1.1-android/5963ed171d561cca6f14659f3439b46a1633ab13/guava-30.1.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protobuf-javalite/3.17.3/a07accfe99ff27e871490cf29fb0d8b0249de320/protobuf-javalite-3.17.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-compat-qual/2.5.5/435dc33e3019c9f019e15f01aa111de9d6b2b79c/checker-compat-qual-2.5.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.7.1/458d9042f7aa6fa9a634df902b37f544e15aacac/error_prone_annotations-2.7.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.3/ba035118bc8bac37d7eff77700720999acd9986d/j2objc-annotations-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/3.13.0/e29afc6adb63a39286d38dc0e00d164117c02f0d/checker-qual-3.13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.8.1/174878f9cc9cbacb8b464d774afb2ccf938f257b/auto-value-annotations-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/7.2/ca2954e8d92a05bacc28ff465b25c70e0f512497/asm-commons-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/7.2/a3ae34e57fa8a4040e28247291d0cc3d6b8c7bcf/asm-util-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/7.2/b6e6abe057f23630113f4167c34bda7086691258/asm-analysis-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/7.2/3a23cc36edaf8fc5a89cb100182758ccb5991487/asm-tree-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/9.1/a99500cf6eea30535eeac6be73899d048f8d12a8/asm-9.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.4/1ee94260f8c51620a35eac33fc1efc01350c751f/pluginapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.4/529649474b53cf8f6f4a483044ade43aebed8a4c/shadowapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.4/70fc5b1699467dfd7de606fc6c02ff9fc1816d9f/annotations-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/0.282/745a7e2f35fdbe6336922e0d492c979dbbfa74fb/sqlite4java-0.282.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.65/320b989112f00a63a3bcfa5a98f31a4f865a20fa/bcprov-jdk15on-1.65.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.19.0/bc51dab3138afbd8517511cfb183dab731d8b1c4/mockito-core-2.19.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.8.10/c3dcdb62b4297eacbe8a763f36044e1f1b04709b/byte-buddy-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.8.10/957933bd13319beb33138494122e8fd1c347bce4/byte-buddy-agent-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/2.6/639033469776fd37c08358c6b92a4761feb2af4b/objenesis-2.6.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi/1.11.0/14c426fc49ee29eb871e8b259326e967082a72ad/moshi-1.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/2.2.2/36f483536153f15339a8b48d508e22be7c9c531a/okio-2.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.istack/istack-commons-runtime/3.0.8/d6a97364045aa6b99bf2d3c566a3f98599c2d296/istack-commons-runtime-3.0.8.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.activation/jakarta.activation-api/1.2.1/562a587face36ec7eff2db7f2fc95425c6602bc1/jakarta.activation-api-1.2.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/txw2/2.3.2/ce5be7da2e442c25ec14c766cb60cb802741727b/txw2-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.xml.fastinfoset/FastInfoset/1.2.16/4eb6a0adad553bf759ffe86927df6f3b848c8bea/FastInfoset-1.2.16.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/res:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/res:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/domain/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/domain/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/domain/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/domain/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/mockwebserver/4.1.0/7269c5f584bef7851fbfb38cd9c9be5959ceae79/mockwebserver-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-junit/1.4.20/e071a8fda380cb9d347a40e920febab251b464e2/kotlin-test-junit-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test/1.4.20/37f4d139996ed0bbd88df805e4df833040431ff6/kotlin-test-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-annotations-common/1.4.20/a82b48a34377ac287c138ea21e09b511ab21c374/kotlin-test-annotations-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-common/1.4.20/2191d43275c9a8558adea84729c1ca372a463e59/kotlin-test-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth.extensions/truth-liteproto-extension/1.1.3/b293a67586f9a39d9f1e0f0ed3174c180ec7070/truth-liteproto-extension-1.1.3.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.20/9de2c79e95d4b4699a455e88ba285a95352e0bea/kotlin-stdlib-jdk7-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.3.4/93dc6aca51305c1b58ccf5a6811665ed58f48ff1/kotlinx-coroutines-android-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.20/9be77b243a362b745e365f286627b8724337009c/kotlin-stdlib-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.20/c6761d7805b5312302f2bbd78cda68c976ce0c70/kotlin-stdlib-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.0.0/42858b26cafdaa69b6149f45dfc2894007bc2c7a/collection-1.0.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/28.1-android/c2526f8fad32a65a6d7032dd8e9524eb276b108b/guava-28.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.3.2/d1a0c5032570e0f64be6b4d9c90cdeb103129029/error_prone_annotations-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.18/f7aa683ea79dc6681ee9fb95756c999acbb62f5d/animal-sniffer-annotations-1.18.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.6.5/c3dad10377f0e2242c9a4b88e9704eaf79103679/auto-value-annotations-1.6.5.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/res:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/res:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/data/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/data/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/data/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/data/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/transforms-3/c0b72f579d9d1fe784b93659eeaa0248/transformed/multidex-instrumentation-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2470781e78212e0a3fca6abd7b631a19/transformed/monitor-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi-kotlin/1.11.0/d8f796170ae085954c9185351a831e276a3d44/moshi-kotlin-1.11.0.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/utility/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/utility/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/utility/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/utility/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/1.0/c949a840a6acbc5268d088e47b04177bf90b3cad/listenablefuture-1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.oppia/androidsvg/6bd15f69caee3e6857fcfcd123023716b4adec1d/58890ad245ff948ce9f19c772dcb88dbf5b8544e/androidsvg-6bd15f69caee3e6857fcfcd123023716b4adec1d.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/disklrucache/4.11.0/ed93d2e20549ad85f692d964788ec77520d78a8b/disklrucache-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/annotations/4.11.0/c57bae5a18147f8ae22f4da49baac875c6b6f84f/annotations-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.12.1/dc6d02e4e68514eff5631963e28ca7742ac69efe/okhttp-3.12.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/1.15.0/bc28b5a964c8f5721eb58ee3f3c47a9bcbf4f4d8/okio-1.15.0.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/res:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/model/build/classes/java/main:/home/ayush/opensource/oppia-android/model/build/resources/main:/home/ayush/opensource/oppia-android/model/build/libs/model.jar:/home/ayush/.gradle/caches/transforms-3/bb2c0feeabbe4ca138704f9cfbac2aab/transformed/room-runtime-2.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9399f18d7867437e16131879faa6bcbf/transformed/jetified-lifecycle-service-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/97dfe740135addea92e170a34026ecc8/transformed/sqlite-framework-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6e04fd8affb200758671fe93167571d0/transformed/sqlite-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.2.5/f5e3b73a0c2ab5e276e26868e4ce3542baede207/room-common-2.2.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-debug/1.2.2/d24ea2861c7b84965b0e395f6c09ffedf27f890d/kotlinx-coroutines-debug-1.2.2.jar:/home/ayush/opensource/oppia-android/domain/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/data/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/model/build/resources/test:/home/ayush/opensource/oppia-android/utility/build/intermediates/java_res/debug/out:/home/ayush/.gradle/caches/transforms-3/c4c90c1865fbc502b00d7cf6afab03c8/transformed/android.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 org.oppia.android.testing.espresso.TextInputActionTest,testTextInputAction_ErrorTextExisted_matchesSafelyTrue
[Robolectric] WARN: Android SDK 29 requires Java 9 (have Java 8). Tests won't be run on SDK 29 unless explicitly requested.
Called loadFromPath(/system/framework/framework-res.apk, true); mode=binary sdk=28

java.lang.Exception: Main looper has queued unexecuted runnables. This might be the cause of the test failure. You might need a shadowOf(getMainLooper()).idle() call.

	at org.robolectric.android.internal.AndroidTestEnvironment.checkStateAfterTestFailure(AndroidTestEnvironment.java:502)
	at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:581)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$0(SandboxTestRunner.java:263)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:89)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: expected to be true
	at org.oppia.android.testing.espresso.TextInputActionTest.testTextInputAction_ErrorTextExisted_matchesSafelyTrue(TextInputActionTest.kt:50)


Process finished with exit code 255
```
But using TextInputActionTestActivity.kt with ActivityScenarioRule fixes that error by simply giving out this in case of a failed test : 
```
/home/ayush/android-studio/jre/bin/java -ea -Didea.test.cyclic.buffer.size=1048576 -javaagent:/home/ayush/android-studio/lib/idea_rt.jar=45199:/home/ayush/android-studio/bin -Dfile.encoding=UTF-8 -classpath /home/ayush/android-studio/lib/idea_rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit5-rt.jar:/home/ayush/android-studio/plugins/junit/lib/junit-rt.jar:/home/ayush/Android/Sdk/platforms/android-29/data/res:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debugUnitTest/classes:/home/ayush/opensource/oppia-android/testing/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/debugUnitTest/R.jar:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debugUnitTest:/home/ayush/opensource/oppia-android/testing/build/intermediates/unit_test_config_directory/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debugUnitTest/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/testing/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/testing/build/tmp/kotlin-classes/debug:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.20/411fc46e908bfa9c034f52b0d31b2e1f61f06127/kotlin-reflect-1.4.20.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0191d205d716f4f112411644b867daa8/transformed/jetified-junit-1.1.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1db8384f1a66ab63c7518e92c9b85f1b/transformed/jetified-core-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/jaxb-runtime/2.3.2/5528bc882ea499a09d720b42af11785c4fc6be2a/jaxb-runtime-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jvnet.staxex/stax-ex/1.8.1/78011e483a21102fb4858f3e8f269a677e50aa23/stax-ex-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.xml.bind/jakarta.xml.bind-api/2.3.2/8d49996a4338670764d7ca4b85a1c4ccf7fe665d/jakarta.xml.bind-api-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.4/418c5bfae392fdbf71cd463a42a3e8c3b839a924/robolectric-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.4/fbcda51d8e6f3a3897ae5cedc7aa481815745290/junit-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.4/3cedd73c5aedaf79fb9a593552816c9fb3282f2/sandbox-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.4/90028766e71353ad6f57d7bcb56ac0d861da18c3/shadows-framework-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.4/a2ee1324bcb62724e6cbfa655bdb5683948a554c/resources-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.4/44c40ac0d2ef1e7c8b0f6c4e224ef26d356170f1/utils-reflector-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.4/9241a3c4bd01627447c76d9b67614808c78ffdd9/plugins-maven-dependency-resolver-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.4/c54b2638d64e7bd4e1e45c4fe8038305402bd711/utils-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.4.20/8f6239ddd98bf9937a81d8a932a185dab930e82a/kotlin-android-extensions-runtime-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-test/1.2.2/93d934029985b6ae62c26cb7fcbdfeecd9b684be/kotlinx-coroutines-test-1.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.10/e2b3c6695eee6085e606d96d685396dce23a3a06/kotlin-reflect-1.4.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.3.4/21598c7ceabbcafb953cd151cd5ad44615a253c0/kotlinx-coroutines-core-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/converter-moshi/2.5.0/95fe2632a019f26d9c53129db192ebc7f3524d35/converter-moshi-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit-mock/2.5.0/3615f8951fd30235331a6027779dfae98b203132/retrofit-mock-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.retrofit2/retrofit/2.5.0/713ce36037bf24a76a3974c05cb85c3f754b1cc3/retrofit-2.5.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/4.1.0/9308be2b5f9174def7d4795f415b60f5632b9fb9/okhttp-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.21/4a668382d7c38688d3490afde93b6a113ed46698/kotlin-stdlib-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.21/7f48a062aa4b53215998780f7c245a4276828e1d/kotlin-stdlib-common-1.4.21.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.2.0-alpha03/4ef09a745007778eef83b92f8f23987a8ea59496/lifecycle-common-2.2.0-alpha03.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.1.0/e3a6fb2f40e3a3842e6b7472628ba4ce416ea4c8/annotation-1.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth/truth/1.1.3/5e2491cc44dce7380165963a24e91a0af4556a17/truth-1.1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.android.apps.common.testing.accessibility.framework/accessibility-test-framework/2.0/7837f95bdff9f797003454247fef02b78d49ae75/accessibility-test-framework-2.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-integration/1.3/5de0c73fef18917cd85d0ab70bb23818685e4dfd/hamcrest-integration-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-library/1.3/4785a3c21320980282f9f33d0d1264a69040538f/hamcrest-library-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.sf.kxml/kxml2/2.3.0/ccbc77a5fd907ef863c29f3596c6f54ffa4e9442/kxml2-2.3.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup/javawriter/2.1.1/67ff45d9ae02e583d0f9b3432a5ebbe05c30c966/javawriter-2.1.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.dagger/dagger/2.27/7bee2792b10523e298a7de508635c6eed0c309be/dagger-2.27.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/30.1.1-android/5963ed171d561cca6f14659f3439b46a1633ab13/guava-30.1.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protobuf-javalite/3.17.3/a07accfe99ff27e871490cf29fb0d8b0249de320/protobuf-javalite-3.17.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-compat-qual/2.5.5/435dc33e3019c9f019e15f01aa111de9d6b2b79c/checker-compat-qual-2.5.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.7.1/458d9042f7aa6fa9a634df902b37f544e15aacac/error_prone_annotations-2.7.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.3/ba035118bc8bac37d7eff77700720999acd9986d/j2objc-annotations-1.3.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/3.13.0/e29afc6adb63a39286d38dc0e00d164117c02f0d/checker-qual-3.13.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.8.1/174878f9cc9cbacb8b464d774afb2ccf938f257b/auto-value-annotations-1.8.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/7.2/ca2954e8d92a05bacc28ff465b25c70e0f512497/asm-commons-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/7.2/a3ae34e57fa8a4040e28247291d0cc3d6b8c7bcf/asm-util-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/7.2/b6e6abe057f23630113f4167c34bda7086691258/asm-analysis-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/7.2/3a23cc36edaf8fc5a89cb100182758ccb5991487/asm-tree-7.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/9.1/a99500cf6eea30535eeac6be73899d048f8d12a8/asm-9.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.4/1ee94260f8c51620a35eac33fc1efc01350c751f/pluginapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.4/529649474b53cf8f6f4a483044ade43aebed8a4c/shadowapi-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.4/70fc5b1699467dfd7de606fc6c02ff9fc1816d9f/annotations-4.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/0.282/745a7e2f35fdbe6336922e0d492c979dbbfa74fb/sqlite4java-0.282.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.65/320b989112f00a63a3bcfa5a98f31a4f865a20fa/bcprov-jdk15on-1.65.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.19.0/bc51dab3138afbd8517511cfb183dab731d8b1c4/mockito-core-2.19.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.8.10/c3dcdb62b4297eacbe8a763f36044e1f1b04709b/byte-buddy-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.8.10/957933bd13319beb33138494122e8fd1c347bce4/byte-buddy-agent-1.8.10.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/2.6/639033469776fd37c08358c6b92a4761feb2af4b/objenesis-2.6.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi/1.11.0/14c426fc49ee29eb871e8b259326e967082a72ad/moshi-1.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/2.2.2/36f483536153f15339a8b48d508e22be7c9c531a/okio-2.2.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.istack/istack-commons-runtime/3.0.8/d6a97364045aa6b99bf2d3c566a3f98599c2d296/istack-commons-runtime-3.0.8.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/jakarta.activation/jakarta.activation-api/1.2.1/562a587face36ec7eff2db7f2fc95425c6602bc1/jakarta.activation-api-1.2.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.glassfish.jaxb/txw2/2.3.2/ce5be7da2e442c25ec14c766cb60cb802741727b/txw2-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.sun.xml.fastinfoset/FastInfoset/1.2.16/4eb6a0adad553bf759ffe86927df6f3b848c8bea/FastInfoset-1.2.16.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/777973247db84021367655288a04537d/transformed/jetified-lifecycle-livedata-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f7cefa9e4fe1d4547d41c40fec0b835a/transformed/jetified-core-ktx-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1eff5eb538f00ca943a8e1591a8cacf1/transformed/jetified-konfetti-1.2.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/26d0b497347908d3577871c97601ae37/transformed/jetified-lifecycle-livedata-core-ktx-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/75e3b061b2ce69a4718f851b7c63239a/transformed/jetified-core-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f1e27a88ba9b2a74a3cafe7e2f459a73/transformed/espresso-accessibility-3.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/866d1b6688247226e99f88270de21d9c/transformed/espresso-core-3.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8bd007dccfb9fdf61bbef532406fb708/transformed/runner-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/res:/home/ayush/.gradle/caches/transforms-3/69b30c49e9b02512b96ded49f085d0f7/transformed/material-1.2.0-alpha02/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/a97f286ff0ff3ad2b76319d23c3f7e04/transformed/appcompat-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/6c8f29a2faf5346a24ee74910b26ef78/transformed/coordinatorlayout-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/24977add424b0a399da07986526bbd5f/transformed/jetified-viewpager2-1.0.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7f2f61ec5bc548e736b6baffcc6d8b96/transformed/fragment-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/res:/home/ayush/.gradle/caches/transforms-3/0e842be7ef9cd0e8f76f78d6dee4df3d/transformed/recyclerview-1.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/res:/home/ayush/.gradle/caches/transforms-3/56821d40195eb2c481d8b1c40327469f/transformed/transition-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/27b5ee5f1640d5aa21cb787f3cd1c39e/transformed/jetified-appcompat-resources-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54ca83273b8b6a9a2719a9af10d619fe/transformed/vectordrawable-animated-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e7d52aeb681a5c3d210357c109e60b75/transformed/vectordrawable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/698fbd0f97fe90d73f79a343fb84f0f9/transformed/drawerlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f5ad73e0f07436d8fccfa9643e9b1b2/transformed/viewpager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/808acb00205f854ea585c37077c0cd74/transformed/customview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/44113c030efdc5698ce44480c0e8dfd1/transformed/loader-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a1c1f386c52fc3375910ae07f1015c5c/transformed/jetified-activity-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/res:/home/ayush/.gradle/caches/transforms-3/e09a8c85d2ee35f3888efe726b003617/transformed/core-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4f584bd434c2aeff376230c82b880943/transformed/monitor-1.3.0-rc03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c37cafd6aab9541349864f378a2b3bf0/transformed/lifecycle-runtime-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/af61258b27c5e541849f0be7fb4d34e6/transformed/lifecycle-livedata-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/5bbb38904986117e11bb949f43fff133/transformed/lifecycle-livedata-core-2.2.0-alpha03/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/000f077f4ba1091be80387cf63e3cf81/transformed/jetified-savedstate-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/c80e10f4e75a95dc21eb81479cf1d4e6/transformed/cardview-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/df33c5e49cc35cd7c7deca158ccfe52f/transformed/cursoradapter-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c171b660bdf74a0fd749cbc43cc7c4ca/transformed/versionedparcelable-1.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c552bb35cadc9455d68d10159b9eacac/transformed/core-runtime-2.1.0-rc01/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1f4556be9519c87322ef4d7c0892d131/transformed/lifecycle-viewmodel-2.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/cb924704fb2ef83cdcd7a30328174f2f/transformed/interpolator-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2690b7188e5d21bcb51e7b5fdf2a47d2/transformed/espresso-idling-resource-3.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/domain/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/domain/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/domain/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/domain/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/mockwebserver/4.1.0/7269c5f584bef7851fbfb38cd9c9be5959ceae79/mockwebserver-4.1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-junit/1.4.20/e071a8fda380cb9d347a40e920febab251b464e2/kotlin-test-junit-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test/1.4.20/37f4d139996ed0bbd88df805e4df833040431ff6/kotlin-test-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-annotations-common/1.4.20/a82b48a34377ac287c138ea21e09b511ab21c374/kotlin-test-annotations-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-test-common/1.4.20/2191d43275c9a8558adea84729c1ca372a463e59/kotlin-test-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.truth.extensions/truth-liteproto-extension/1.1.3/b293a67586f9a39d9f1e0f0ed3174c180ec7070/truth-liteproto-extension-1.1.3.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9fa491c7b1d05ff33e6c9d83874339a7/transformed/work-testing-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/76b0942a2c5f387b86ddb674c8ca2af3/transformed/core-testing-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.20/9de2c79e95d4b4699a455e88ba285a95352e0bea/kotlin-stdlib-jdk7-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.3.4/93dc6aca51305c1b58ccf5a6811665ed58f48ff1/kotlinx-coroutines-android-1.3.4.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.20/9be77b243a362b745e365f286627b8724337009c/kotlin-stdlib-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.20/c6761d7805b5312302f2bbd78cda68c976ce0c70/kotlin-stdlib-common-1.4.20.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.0.0/42858b26cafdaa69b6149f45dfc2894007bc2c7a/collection-1.0.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/28.1-android/c2526f8fad32a65a6d7032dd8e9524eb276b108b/guava-28.1-android.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.3.2/d1a0c5032570e0f64be6b4d9c90cdeb103129029/error_prone_annotations-2.3.2.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.18/f7aa683ea79dc6681ee9fb95756c999acbb62f5d/animal-sniffer-annotations-1.18.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.6.5/c3dad10377f0e2242c9a4b88e9704eaf79103679/auto-value-annotations-1.6.5.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/677b01921371323f334f30a08422e0b7/transformed/jetified-firebase-crashlytics-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/efd8736f3d88d15083599549db74971e/transformed/work-runtime-ktx-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0b5515c6d1b41818b51b2bcb2a690eb3/transformed/jetified-firebase-analytics-ktx-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f9f2be8ea0140f46017909198e724864/transformed/jetified-firebase-common-ktx-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/res:/home/ayush/.gradle/caches/transforms-3/386d73d79363e4e5aa70d9a70b27bf60/transformed/appcompat-1.0.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e52f40dabb45643687e4ec1ccc79c884/transformed/exifinterface-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3ce28c4c5f16eeab97dbc331e0994604/transformed/jetified-firebase-iid-20.1.5/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/49652581b1103c8bd376e04a6c3449b2/transformed/jetified-firebase-analytics-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1c36c4f1770a99eebd68d00a81498b5e/transformed/jetified-play-services-measurement-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1fc4d0b19c5132512c5bce35841c51a/transformed/jetified-firebase-installations-16.3.2/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/1469e90ea9f18dca987670cada7ffab1/transformed/jetified-firebase-common-19.3.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c93843dd4a1153060aefffcfc2e779bf/transformed/jetified-firebase-installations-interop-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f3cc86ad4f43a5a5ca97c5b11b9f60f5/transformed/jetified-firebase-iid-interop-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/a30e34e4feccce471956097bb45fa9f6/transformed/jetified-play-services-base-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ed9c4a23cc2b3517c5848dba8b0eaf8/transformed/jetified-play-services-tasks-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3c9ee099af92b05ecff6b7b8c221c98e/transformed/jetified-firebase-measurement-connector-18.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/c0452c45ca43cb7313b1e9d5efc3be78/transformed/jetified-play-services-measurement-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2718baced76b247f04d343482edd43a2/transformed/jetified-play-services-measurement-sdk-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/06c332e334dc2f31caa19348a77fd5da/transformed/jetified-play-services-measurement-impl-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/3770ae673c3540b512df36f04a11362b/transformed/jetified-play-services-stats-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/ddbb056add593fdc4d637ad00828fe1f/transformed/jetified-play-services-ads-identifier-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0ded12faa12f705b260a0afd5520742f/transformed/jetified-play-services-measurement-sdk-api-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/aa2b2e73a5ffa45ccee7c19919f13bec/transformed/jetified-play-services-measurement-base-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/res:/home/ayush/.gradle/caches/transforms-3/19a9d679698f068f95e8f3eea8f6d23c/transformed/jetified-play-services-basement-17.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4297da355309cca83a6661e4fc248d98/transformed/fragment-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/671495411e2e65f28314a7fb1cc31e26/transformed/vectordrawable-animated-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9adcff5d603db3a3bfdf59cff43639fc/transformed/legacy-support-core-ui-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe6a0bc8c83d39db61f99fe20470b80a/transformed/legacy-support-core-utils-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/72ad1f717dad022b82d81ec4d1c6c049/transformed/vectordrawable-1.0.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/res:/home/ayush/.gradle/caches/transforms-3/995daff4b48f4d3ad18ce71f4dca3187/transformed/coordinatorlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/2f7b62cd739584b25df5e27a5b29b7ee/transformed/slidingpanelayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6b5e34e9268af3900b6a9c2dce2d3864/transformed/swiperefreshlayout-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/66d141643b43c7deb158710acc35b50c/transformed/asynclayoutinflater-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/bfd62c20dd093978ee2595a43b00d011/transformed/jetified-firebase-components-16.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/facd70070d21fca9722dc8637de36489/transformed/jetified-transport-backend-cct-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/54594c5a4e33227f1f1618a0730fc8fe/transformed/jetified-transport-runtime-2.2.1/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b8af66722fecf8b6dffded469838a7b9/transformed/jetified-firebase-encoders-json-16.1.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/fe2cd982c989d4976866271caccd6ee5/transformed/documentfile-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/8449b54376369b8114e6e01dd0743c74/transformed/localbroadcastmanager-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/b1830a2c9ab00b8810c39a8deeb0c86b/transformed/print-1.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/77ff0bcc4391e7fb37baec41c13b396c/transformed/lifecycle-viewmodel-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/res:/home/ayush/.gradle/caches/transforms-3/d0593da85ed429c13a2ca2f0bedfa110/transformed/work-runtime-2.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/0f01fc3f3b0d745d4bf182e6e8de7bf6/transformed/jetified-transport-api-2.2.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/data/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/data/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/opensource/oppia-android/data/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/data/build/tmp/kapt3/classes/debug:/home/ayush/.gradle/caches/transforms-3/c0b72f579d9d1fe784b93659eeaa0248/transformed/multidex-instrumentation-2.0.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/2470781e78212e0a3fca6abd7b631a19/transformed/monitor-1.2.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi-kotlin/1.11.0/d8f796170ae085954c9185351a831e276a3d44/moshi-kotlin-1.11.0.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a855b7bcac159e6741e143e84a2f5619/transformed/multidex-2.0.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/utility/build/intermediates/javac/debug/classes:/home/ayush/opensource/oppia-android/utility/build/tmp/kapt3/classes/debug:/home/ayush/opensource/oppia-android/utility/build/tmp/kotlin-classes/debug:/home/ayush/opensource/oppia-android/utility/build/intermediates/compile_only_not_namespaced_r_class_jar/debug/R.jar:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/a982c9f206a8cc6b3f16a50db9859438/transformed/jetified-truth-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/e386c0056882f499a94e11edb0454d7d/transformed/jetified-core-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/4635b6a3308d542f2902459b379e7553/transformed/monitor-1.4.0/AndroidManifest.xml:/home/ayush/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/1.0/c949a840a6acbc5268d088e47b04177bf90b3cad/listenablefuture-1.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.oppia/androidsvg/6bd15f69caee3e6857fcfcd123023716b4adec1d/58890ad245ff948ce9f19c772dcb88dbf5b8544e/androidsvg-6bd15f69caee3e6857fcfcd123023716b4adec1d.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/disklrucache/4.11.0/ed93d2e20549ad85f692d964788ec77520d78a8b/disklrucache-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.github.bumptech.glide/annotations/4.11.0/c57bae5a18147f8ae22f4da49baac875c6b6f84f/annotations-4.11.0.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.12.1/dc6d02e4e68514eff5631963e28ca7742ac69efe/okhttp-3.12.1.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/1.15.0/bc28b5a964c8f5721eb58ee3f3c47a9bcbf4f4d8/okio-1.15.0.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/res:/home/ayush/.gradle/caches/transforms-3/7703d997e890914e3d1a7515f5808f49/transformed/jetified-glide-4.11.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/585817f784310131073f16f49c96abd1/transformed/jetified-firebase-core-17.5.0/AndroidManifest.xml:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/f825dea07cae0a53ace3f28b70d0afec/transformed/jetified-gifdecoder-4.11.0/AndroidManifest.xml:/home/ayush/opensource/oppia-android/model/build/classes/java/main:/home/ayush/opensource/oppia-android/model/build/resources/main:/home/ayush/opensource/oppia-android/model/build/libs/model.jar:/home/ayush/.gradle/caches/transforms-3/bb2c0feeabbe4ca138704f9cfbac2aab/transformed/room-runtime-2.2.5/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/9399f18d7867437e16131879faa6bcbf/transformed/jetified-lifecycle-service-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/97dfe740135addea92e170a34026ecc8/transformed/sqlite-framework-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/transforms-3/6e04fd8affb200758671fe93167571d0/transformed/sqlite-2.1.0/jars/classes.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.2.5/f5e3b73a0c2ab5e276e26868e4ce3542baede207/room-common-2.2.5.jar:/home/ayush/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-debug/1.2.2/d24ea2861c7b84965b0e395f6c09ffedf27f890d/kotlinx-coroutines-debug-1.2.2.jar:/home/ayush/opensource/oppia-android/domain/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/data/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/testing/build/intermediates/java_res/debug/out:/home/ayush/opensource/oppia-android/model/build/resources/test:/home/ayush/opensource/oppia-android/utility/build/intermediates/java_res/debug/out:/home/ayush/.gradle/caches/transforms-3/c4c90c1865fbc502b00d7cf6afab03c8/transformed/android.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 -junit4 org.oppia.android.testing.espresso.TextInputActionTest,testTextInputAction_ErrorTextExisted_matchesSafelyTrue
[Robolectric] WARN: Android SDK 29 requires Java 9 (have Java 8). Tests won't be run on SDK 29 unless explicitly requested.
Called loadFromPath(/system/framework/framework-res.apk, true); mode=binary sdk=28

expected to be true

	at org.oppia.android.testing.espresso.TextInputActionTest.testTextInputAction_ErrorTextExisted_matchesSafelyTrue(TextInputActionTest.kt:49)


Process finished with exit code 255
```
Hence, I decided to keep TextInputActionTestActivity's use for the Test.

Reply:
Ah I see--shouldn't you be removing the test activity & the manifest outright if it's not needed? I'm actually a little confused here because it seems like the activity is partly being used, but partly not. What is the actual solution that works & why? I think that needs to be figured out.

Beyond that, I also suggest getting just one test passing for both Bazel & Gradle before trying to finish the rest of the tests. The environments are slightly different from one another, so it's important to get a new test suite pattern like this working on both, first.

Reply:
Per my comment above, it seems like the activity is actually needed but I'm still not clear on this (since there seems to also be a typo in the manifest).